std/random.d

```diff
@@ -1299,19 +1373,59 @@ random number sequences every run.
 Returns:
 A single unsigned integer seed value, different on each successive call
 */
-@property uint unpredictableSeed() @trusted nothrow @nogc
+alias unpredictableSeed = unpredictableSeedOf!uint;
+/// ditto
+@property UIntType unpredictableSeedOf(UIntType)() @nogc nothrow @trusted
```

Comment:
As this is a new public symbol it would require a changelog entry + @andralex approval.
Maybe it's easier to set it to `private` for now?

Reply:
Since you have more experience here, if you think splitting it will speed along the review process I'll do that and amend the title.

Reply:
> f you think splitting it will speed along the review process

Yeah I do think it will help. Approval of a new public symbol takes usually > one month (as once its added it can't be removed or modified anymore), so I would recommend setting it to `private` and once this PR is merged, opening a PR that just removes `private`. With this approach, you aren't blocked on the approval for new symbols.

Reply:
AFAIK a public alias of a private member doesn't work so I instead removed the `unpredictableSeedOf!UIntType` template.